### PR TITLE
[MyNice] Addition of Courtesy Light Channel

### LINKED ISSUE #14795: [Mynice] Warnings generated when operating Bridge
When disabling IT4Wifi bridge we can observe : 

`
2023-04-13 11:07:01.625 [WARN ] [core.thing.internal.ThingManagerImpl] - Disposing handler for thing 'mynice:it4wifi:bridge' takes more than 5000ms.
`

From time to time also this one : 

`
2023-04-13 11:07:54.543 [WARN ] [.core.thing.binding.BaseThingHandler] - Handler It4WifiHandler tried updating the thing status although the handler was already disposed.
`

I also see that enabling / disable thing idles at the third time it is done.

## Expected Behavior
No warning should be displayed

## Steps to Reproduce (for Bugs)
1. Disable / Enable IT4Wifi thing multiple times

### PR DESCRIPTION
**Pull request by clinique**
Gate status channel is now R/W, accepting StopMoveType commands.
Courtesy light channel added.

Also resolves #14795

### CODE REVIEW

File: bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties [hunk 1]
```diff
@@ -31,22 +31,70 @@ thing-type.config.mynice.swing.id.description = ID of the gate on the TP4 bus co
...
+
+# channel types
+
+channel-type.mynice.doorstatus.state.option.open = Open
+channel-type.mynice.doorstatus.state.option.closed = Closed
+channel-type.mynice.doorstatus.state.option.opening = Opening
+channel-type.mynice.doorstatus.state.option.closing = Closing
+channel-type.mynice.doorstatus.state.option.stopped = Stopped
+channel-type.mynice.doorstatus.command.option.stop = Stop
+channel-type.mynice.doorstatus.command.option.move = Move
```
Comment (lolodomo) [inline bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties:91]: This block has to be removed here.

File: bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties [hunk 2]
```diff
@@ -31,22 +31,70 @@ thing-type.config.mynice.swing.id.description = ID of the gate on the TP4 bus co
...
+
+# channel types
+
+channel-type.mynice.command.state.option.stop = Stop
+channel-type.mynice.command.state.option.open = Open
+channel-type.mynice.command.state.option.close = Close
 channel-type.mynice.t4command.state.option.MDAx = Step by Step
 channel-type.mynice.t4command.state.option.MDAy = Stop (as remote control)
 channel-type.mynice.t4command.state.option.MDAz = Open (as remote control)
```
Comment (lolodomo) [inline bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties:100]: This block with command and t4command entries has to be removed here.

File: bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml [hunk 3]
```diff
@@ -47,6 +47,7 @@
 			<channel id="moving" typeId="moving"/>
 			<channel id="command" typeId="command"/>
 			<channel id="t4command" typeId="t4command"/>
+			<channel id="courtesy" typeId="courtesy"/>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml:50]: You need to add update instructions for this new channel.

File: bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml [hunk 4]
```diff
@@ -73,6 +74,7 @@
...
```
Comment (lolodomo) [inline bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml:77]: You need to add update instructions for this new channel.

**Review by lolodomo — CHANGES_REQUESTED** [review]
Review part 1 of 2 (all except handler classes)

File: bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml (hunk 4 above)
Comment (clinique) [reply bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml:77]: Ok, done.

File: bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml (hunk 3 above)
Comment (clinique) [reply bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml:50]: Ok, done

File: bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties:91]: Clean and reruned properties generation.

File: bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties (hunk 2 above)
Comment (clinique) [reply bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/i18n/mynice.properties:100]: Clean and reruned properties generation.

**Comment by clinique** [comment]
@lolodomo, @jlaur  : could I have another code review ?

**Comment by lolodomo** [comment]
> @lolodomo, @jlaur : could I have another code review ?

I will try to find time before the weekend to finish my review.

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**